travis/travis_install_nightly

```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb # downloading updated chrome dev tools
```

Comment:
This should be located just below [this line](https://github.com/Vauxoo/maintainer-quality-tools/blob/819e9b7dc378c96306858c1c44a6b5deb3115289/travis/travis_install_nightly#L139)